modules/markdown/markdown.tsx

```diff
@@ -22,13 +23,20 @@ import {Link} from './link'
 import {Image} from './image'
 import {List, ListItem} from './list'
 
+const styles = StyleSheet.create({
+	// eslint-disable-next-line react-native/no-color-literals
```

Comment:
I don't know why line 27 is the right spot for this comment, when it affects line 31 or something, but I've learned not to question ESLint.

Reply:
Do we want to instead import c.black from frogpond/colors?